CHANGELOG.md

```diff
@@ -21,4 +21,5 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
  - LLM Judgment Processor Improvement ([#27](https://github.com/opensearch-project/search-relevance/pull/27))
  - Deal with experiment processing when no experiment variants exist. ([#45](https://github.com/opensearch-project/search-relevance/pull/45))
 
+
```

Comment:
I guess a line what was changed in the demo script would be nice here instead of an empty line :)